packages/nodes-base/package.json

```diff
@@ -16,13 +16,17 @@
   "types": "dist/src/index.d.ts",
   "scripts": {
     "dev": "npm run watch",
-    "build": "tsc && gulp build:icons && gulp build:translations",
+    "build": "npm-run-all build:esbuild build:assets build:dts",
```

Comment:
Not very familiar with `npm-run-all`, can we use `--parallel` to build faster?

Reply:
Shaved off another second! Thanks for the suggestion! :)